app/src/main/java/com/github/se/eventradar/ui/login/LoginScreen.kt

```diff
@@ -33,17 +46,35 @@ import com.github.se.eventradar.ui.navigation.NavigationActions
 import com.github.se.eventradar.ui.navigation.Route
 
 @Composable
-fun LoginScreen(navigationActions: NavigationActions) {
+fun ErrorDialogBox(openErrorDialog: MutableState<Boolean>) {
+  val display by openErrorDialog
+  if (display == true) {
+    AlertDialog(
+        icon = { Icon(Icons.Default.AccountCircle, contentDescription = "Account Icon") },
+        text = { Text("Sign in Failed. Please try again.", textAlign = TextAlign.Center) },
+        title = { Text("Sign in Failed") },
+        onDismissRequest = { openErrorDialog.value = false },
+        confirmButton = {
+          TextButton(onClick = { openErrorDialog.value = false }) { Text("Confirm") }
+        })
+  }
+}
 
+@SuppressLint("UnrememeredMutableState", "UnrememberedMutableState")
+@Composable
+fun LoginScreen(navigationActions: NavigationActions) {
+  val openErrorDialog = remember { mutableStateOf(false) }
```

Comment:
The standard way to do this is using the `by` keyword (then you won't need that `@SuppressLint` annotation.

Reply:
DM'ed, and solution is ok it turns out 👍